exchanges/bybit/bybit_websocket.go

```diff
@@ -523,7 +524,7 @@ func (by *Bybit) wsProcessLeverageTokenKline(assetType asset.Item, resp *Websock
 	if err != nil {
 		return err
 	}
-	cp, err := currency.NewPairFromString(topicSplit[2])
+	cp, err := by.MatchSymbolWithAvailablePairs(topicSplit[2], assetType, true)
```

Comment:
The only time I believe bybit sends pair data with delimiters is options
and we have this code at present:
```
for _, a := range []asset.Item{asset.CoinMarginedFutures, asset.USDTMarginedFutures, asset.USDCMarginedFutures, asset.Options} {
		if err := by.DisableAssetWebsocketSupport(a); err != nil {
			log.Errorln(log.ExchangeSys, err)
		}
	}
```
eg
```
[DEBUG] | WEBSOCKET | 08/11/2024 11:22:01 | Bybit wss://stream.bybit.com/v5/public/spot: Message received: {"topic":"orderbook.50.BTCUSDT","ts":1731025321625,"type":"delta","data":{"s":"BTCUSDT","b":[["76279.57","0.015295"],["76272.82","0.028297"],["76272.1","0"],["76271.98","0"]],"a":[["76293.07","0.015573"],["76298.47","0"]],"u":2491950,"seq":47455557170},"cts":1731025321618}
```

So unless you've seen otherwise, it can be set to `false`

Reply:
I can but its just going to switch back when #1670 gets merged and also the traversal doesn't add much in over head.

Still need me to do it?

Reply:
Isolating just the rune traversal, it adds roughtly 1ns per character. The longest spot pair is 10 characters, so I'm cool if you're cool with that. I would encourage you to consider something for that PR though if only because silly ones like `10000000AIDOGEUSDT`

Reply:
[fixed](https://github.com/thrasher-corp/gocryptotrader/pull/1680/commits/0726e858facc82d4e3015eb5cb3ca952cbccdb57)